lib/pages/forward/forward_view.dart

```diff
@@ -27,59 +27,172 @@ class ForwardView extends StatelessWidget {
-                    );
-                  }
+      body: Column(
+        children: [
+          Expanded(
+            child: SingleChildScrollView(
+              padding:
+                  const EdgeInsetsDirectional.all(ForwardViewStyle.paddingBody),
+              child: Column(
+                children: [
+                  const RecentChatsTitle(),
+                  ValueListenableBuilder<List<Room>>(
+                    valueListenable: controller.recentlyChatsNotifier,
+                    builder: (context, rooms, child) {
+                      if (rooms.isNotEmpty) {
+                        return RecentChatList(
+                          rooms: rooms,
+                          selectedChatNotifier:
+                              controller.selectedRoomIdNotifier,
+                          onSelectedChat: (roomId) =>
+                              controller.onToggleSelectChat(roomId),
+                          recentChatScrollController:
+                              controller.recentChatScrollController,
+                        );
+                      }
 
-                  return const SizedBox.shrink();
-                },
+                      return const SizedBox.shrink();
+                    },
+                  ),
+                ],
               ),
-            ],
+            ),
           ),
-        ),
+          if (!controller.isFullScreen)
+            _WebActionsButton(
+              selectedChatNotifier: controller.selectedRoomIdNotifier,
+              forwardMessageNotifier: controller.forwardMessageNotifier,
+              forwardAction: controller.forwardAction,
+            ),
+        ],
       ),
-      floatingActionButton: ForwardButton(
-        forwardAction: controller.forwardAction,
-        selectedChatNotifier: controller.selectedRoomIdNotifier,
-        forwardMessageNotifier: controller.forwardMessageNotifier,
+      floatingActionButton: controller.isFullScreen
+          ? _ForwardButton(
+              forwardAction: controller.forwardAction,
+              selectedChatNotifier: controller.selectedRoomIdNotifier,
+              forwardMessageNotifier: controller.forwardMessageNotifier,
+            )
+          : null,
+    );
+  }
+}
+
+class _WebActionsButton extends StatelessWidget {
+  final ValueNotifier<String> selectedChatNotifier;
+
+  final ValueNotifier<Either<Failure, Success>?> forwardMessageNotifier;
+
+  final void Function() forwardAction;
+
+  const _WebActionsButton({
+    // ignore: unused_element
+    super.key,
+    required this.selectedChatNotifier,
+    required this.forwardMessageNotifier,
+    required this.forwardAction,
+  });
+
+  @override
+  Widget build(BuildContext context) {
+    return Padding(
+      padding: ForwardViewStyle.webActionsButtonPadding,
+      child: Row(
+        mainAxisAlignment: MainAxisAlignment.end,
+        children: [
+          TwakeTextButton(
+            onTap: () => context.pop(),
```

Comment:
context.pop() may not work in web

Reply:
It works as expected:

[Capture vidéo du 09-04-2024 09:48:03.webm](https://github.com/linagora/twake-on-matrix/assets/31937920/ad209020-8786-485a-8e8d-e23bdaa41d4f)